.github/workflows/wildfly-it-tests-manual-bootable.yml

```diff
@@ -0,0 +1,84 @@
+name: WildFly Integration Tests (Bootable JAR) - Manual execution
+
+on:
+  workflow_dispatch:
```

Comment:
Is there some option how to trigger the CI run from PR (eg by comment) directly?

Reply:
Those are separate tasks. 
With a first one, we created automated CI checks for PR `push` events, see
With this PR we create a manual execution workflow, similar to a face job.
If we want to handle comment based events, then IMO we should do that in a separate work unit. WDYT?

Reply:
+1